functions.lua

```diff
@@ -574,116 +592,134 @@ function tp.tpc_send(sender, coordinates)
 	end
 end
 
-function tp.tpr_deny(name)
-	if not tp.tpr_list[name] and not tp.tphr_list[name]
-	and not tp.tpc_list[name] and not tp.tpn_list[name] then
-		minetest.chat_send_player(name, S("Usage: /tpn allows you to deny teleport/area requests sent to you by other players."))
-		if minetest.get_modpath("chat2") then
-			chat2.send_message(minetest.get_player_by_name(name), S("Usage: /tpn allows you to deny teleport/area requests sent to you by other players."), 0xFFFFFF)
+function tpr.tpr_deny(name)
+	local no_areas_message = "Usage: /tpn allows you to deny teleport requests sent to you by other players."
+	local areas_message = "Usage: /tpn allows you to deny teleport/area requests sent to you by other players."
+
+	if not tpr.tpr_list[name] and not tpr.tphr_list[name]
+	and not tpr.tpc_list[name] and not tpr.tpn_list[name] then
+		if check_area_requests then
+			minetest.chat_send_player(name, S(areas_message))
+			if minetest.get_modpath("chat2") then
+				chat2.send_message(minetest.get_player_by_name(name), S(areas_message), 0xFFFFFF)
```

Comment:
`S()` must contain a literal string. The translation parser script can only do static text analysis.